2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,11 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
```

Comment:
There should be a variable defined here containing 5 recipe objects. Then there should be some code that loops through those recipes and prints them out as described in the comment.